<?xml version="1.0"?>
<launch>

    <rosparam file="$(find bm_generator)/config/cfg.yaml" command="load" />

    <node pkg="bm_generator" type="bm_generator" name="bm_generator" output="screen" />

</launch>